### PR TITLE
Fix GH actions

### PR DESCRIPTION
**Pull request by gmponos**

### CODE REVIEW

**Comment by gmponos** [comment]
Hi.. I fixed the GH actions now the CI is passing again can you have a look?

File: .github/workflows/static.yml [hunk 1]
```diff
@@ -18,10 +18,10 @@ jobs:
       - name: Install
         uses: docker://composer
         with:
-          args: install --ignore-platform-reqs
+          args: install --ignore-platform-reqs --no-ansi --no-suggest
 
       - name: PHPStan
-        uses: docker://oskarstark/phpstan-ga
+        uses: OskarStark/phpstan-ga@0.12.28
```
Comment (sagikazarmark) [inline .github/workflows/static.yml:24]: Just some nitpicking, but I think this could stay `docker://oskarstark/phpstan-ga`

Comment (gmponos) [reply .github/workflows/static.yml]: This is not working :/ check the last build

Comment (gmponos) [reply .github/workflows/static.yml]: fixed i had a typo

**Comment by frederikbosch** [comment]
I don't know why the docblocks have been removed, and I don't have a lot experience with Github Actions, and I don't know what the benefit is exactly, but I am not against it.

**Comment by gmponos** [comment]
The docblocks have been removed due to php-cs-fixer newest version.. I guess this rule is include inside the `@symfony` ruleset.

**Comment by gmponos** [comment]
https://github.com/moneyphp/money/blob/master/.php_cs.dist#L5

**Comment by frederikbosch** [comment]
Ok, let's merge it in then.

**Pull request merged**